Podcast/EmptyStateView.swift

```diff
@@ -53,7 +53,7 @@ enum EmptyStateType {
         case .pastSearch:
             return "Find your favorite podcast episodes, series, & friends."
         case .bookmarks:
-            return "You can save podcast episodes for later here. Start looking now!"
+            return "" //You can save podcast episodes for later here. Start looking now!"
```

Comment:
why is there no empty state text for bookmarks?

Reply:
It looked weird with the null state and jump back in .. I'm adding in the empty state back in but removing the image so it looks better.